source/opt/type_manager.cpp

```diff
@@ -398,7 +398,6 @@ uint32_t TypeManager::FindPointerToType(uint32_t type_id,
   }
 
   // Must create the pointer type.
-  // TODO(1841): Handle id overflow.
```

Comment:
Why does your change remove these TODOs?

Reply:
Not sure why this was eliminated -- perhaps a merge that didn't happen correctly. Will investigate and add back the TODOs